Source/WebCore/platform/graphics/transforms/PerspectiveTransformOperation.cpp

```diff
@@ -75,10 +74,9 @@ Ref<TransformOperation> PerspectiveTransformOperation::blend(const TransformOper
     }
 
     double pInverse = WebCore::blend(fromPInverse, toPInverse, context);
-    std::optional<Length> p;
-    if (pInverse > 0.0 && std::isnormal(pInverse)) {
-        p = Length(1.0 / pInverse, LengthType::Fixed);
-    }
+    std::optional<float> p;
+    if (pInverse > 0.0 && std::isnormal(pInverse))
+        p = 1.0 / pInverse;
```

Comment:
Not new: I hope that “compute as `double` and then convert to `float`” is a good strategy. Generally nervous about mixing floating point types like this.